src/Modules/Data/Commands.Data/MicrosoftPowerBIMgmt.Data.psd1

```diff
@@ -71,7 +71,7 @@ PowerShellVersion = '3.0'
 FunctionsToExport = '*'
 
 # Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
-CmdletsToExport = 'Get-PowerBIDataset', 'Get-PowerBIDatasource', 'Get-PowerBITable'
+CmdletsToExport = 'Add-PowerBIDataset','New-PowerBIDataset','New-PowerBITable','New-PowerBIColumn','Get-PowerBIDataset', 'Get-PowerBIDatasource', 'Get-PowerBITable'
```

Comment:
Can you update this PR to remove the changes for the New-PowerBIDataset, New-PowerBITable, and New-PowerBIColumn cmdlets for now? I took a look at those files and they look incomplete so I don't think it makes since to merge them yet.

Reply:
These are the completed ones and mandatory cmdlets to use it for Add-PowerBIDataset. Could you give me a bit more detail where it seems to be incomplete?

Reply:
Ah, I misunderstood the purpose of your "New-" cmdlets. I thought they were going to hit the APIs directly to create those objects in Power BI.

Reply:
Thanks, then I leave them as they are.